SDDC_FX3/hf103.c

```diff
@@ -24,9 +27,11 @@ void hf103_GpioInitialize()
     ConfGPIOsimpleout (GPIO_DITH);
     ConfGPIOsimpleout (GPIO_RANDO);
     ConfGPIOsimpleout (GPIO_LED_KIT);
+    ConfGPIOsimpleout (GPIO_ATT_LE);
+    ConfGPIOsimpleout (GPIO_ATT_CLK);
+    ConfGPIOsimpleout (GPIO_ATT_DATA);
```

Comment:
shall we fix HF103 in a separate change?

Reply:
Yes. I restore the previous H103 that will manage in a separate change